playbook-website/package.json

```diff
@@ -4,17 +4,27 @@
   "version": "1.0.0",
   "scripts": {
     "lint": "eslint app/ --ext .js,.jsx",
-    "watch": "NODE_OPTIONS=--openssl-legacy-provider ./bin/webpack-dev-server",
```

Comment:
`./run.sh` handles this, but if you want to watch now you have two options which make little sense to add as scripts here:

1. `bin/vite dev` - watches src and compiles immediately
2. `bin/vite build --watch` - builds for production then watches src and compiles immediately to the `dist` folder